src/System.Private.CoreLib/shared/System/Runtime/CompilerServices/TaskAwaiter.cs

```diff
@@ -424,9 +307,19 @@ public ConfiguredTaskAwaitable.ConfiguredTaskAwaiter GetAwaiter()
             // Its layout must remain the same.
 
             /// <summary>The task being awaited.</summary>
-            internal readonly Task m_task;
+#if CORECLR
+            internal
+#else
+            private
+#endif
+            readonly Task m_task; 
             /// <summary>Whether to attempt marshaling back to the original context.</summary>
-            internal readonly bool m_continueOnCapturedContext;
+#if CORECLR
+            internal
+#else
+            private
+#endif
+            readonly bool m_continueOnCapturedContext; 
```

Comment:
Same for these... is there value in adding ifdefs to keep the field private in corert?

Reply:
Agreed. I wanted to also visibly show the diffs for review.